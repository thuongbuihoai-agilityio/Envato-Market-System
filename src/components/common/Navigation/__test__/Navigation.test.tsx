import { render } from '@testing-library/react';
import { Navigation } from '@components/index';
import { MemoryRouter } from 'react-router-dom';

describe('Navigation test case', () => {
  it('should render correctly with the default destination', () => {
    const { container } = render(
      <MemoryRouter>
        <Navigation>Mock navigation</Navigation>
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });

  it('should render correctly with the given destination', () => {
    const { container } = render(
      <MemoryRouter>
        <Navigation destination="/mock">Mock navigation</Navigation>
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
