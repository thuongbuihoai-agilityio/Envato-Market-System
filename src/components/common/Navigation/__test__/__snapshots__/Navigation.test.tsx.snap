// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation test case should render correctly 1`] = `
<div>
  <a
    class="chakra-link css-1st5xrp"
    href="/"
  >
    Mock navigation
  </a>
</div>
`;

exports[`Navigation test case should render correctly with the default destination 1`] = `
<div>
  <a
    class="chakra-link css-1st5xrp"
    href="/"
  >
    Mock navigation
  </a>
</div>
`;

exports[`Navigation test case should render correctly with the given destination 1`] = `
<div>
  <a
    class="chakra-link css-1st5xrp"
    href="/mock"
  >
    Mock navigation
  </a>
</div>
`;
